web/cobrands/buckinghamshire/assets.js

```diff
@@ -200,7 +215,10 @@ fixmystreet.assets.add($.extend(true, {}, defaults, {
             // unless an asset is selected.
             fixmystreet.body_overrides.do_not_send(layer.fixmystreet.body);
             fixmystreet.body_overrides.remove_only_send();
-            if (fixmystreet.assets.selectedFeature()) {
+            if (fixmystreet.body_overrides.get_only_send() === 'Highways England') {
```

Comment:
This never gets called because the only send was just removed the line above. I think it's then covered by the new event, anyway. Should it just return at the start, like the `found` function? I /think/ they can be consistent.

Reply:
Or just removed, if I'm saying it can be consistent with `found` and my later comment there.